test/unittests/Microsoft.ServiceFabric.Services.Remoting.Tests/Microsoft.ServiceFabric.Services.Remoting.Tests.csproj

```diff
@@ -5,7 +5,8 @@
     <ProjectGuid>{AE4034AC-27BC-43B8-9176-068B4794E25E}</ProjectGuid>
     <AssemblyName>Microsoft.ServiceFabric.Services.Remoting.Tests</AssemblyName>
     <RootNamespace>$(AssemblyName)</RootNamespace>
-   <TargetFramework>net6.0</TargetFramework>
+   <TargetFramework>net9.0</TargetFramework>
+   <EnableUnsafeBinaryFormatterSerialization>true</EnableUnsafeBinaryFormatterSerialization>
```

Comment:
Is the EnableUnsafeBinaryFormatterSerialization property required in addition to the compatibility package?